estuary/utils/story.py

```diff
@@ -0,0 +1,473 @@
+# SPDX-License-Identifier: GPL-3.0+
+
+from __future__ import unicode_literals
+
+import abc
+
+from neomodel import db
+
+from estuary.error import ValidationError
+from estuary.models.koji import ContainerKojiBuild, KojiBuild, ModuleKojiBuild
+from estuary.models.bugzilla import BugzillaBug
+from estuary.models.distgit import DistGitCommit
+from estuary.models.errata import Advisory, ContainerAdvisory
+from estuary.models.freshmaker import FreshmakerEvent
+
+
+class BaseStoryManager(object):
+    """A class containing utility methods to create a story for an artifact."""
+
+    @staticmethod
+    def get_story_manager(item, config, limit=False):
+        """
+        Select which story flow to follow.
+
+        :param node item: a Neo4j node whose story is requested by the user
+        :param flask.config.Config config: flask config
+        :kwarg bool limit: specifies if LIMIT keyword should be added to the created cypher query
+        :return: instance of one of the story manager classes
+        :rtype: ModuleStoryManager/ContainerStoryManager
+        """
+        for class_name in config.get('STORY_MANAGER_SEQUENCE', ['ContainerStoryManager']):
+            story_manager_cls = globals().get(class_name)
+            if not story_manager_cls:
+                raise TypeError('')
+            story_manager = story_manager_cls()
+            story_manager.forward_story = story_manager.get_story_nodes(item, limit=limit)
+            story_manager.backward_story = story_manager.get_story_nodes(
+                item, reverse=True, limit=limit)
+            BaseStoryManager.story_flow_list = story_manager.story_flow_list
+            story = story_manager.forward_story + story_manager.backward_story
```

Comment:
I know you said it doesn't matter which order, but I'd prefer if the order was correct in the event `is_valid` is expanded to do more complex checks. That way we can avoid confusion in the future.